vpr/src/draw/draw.cpp

```diff
@@ -326,7 +326,8 @@ static void initial_setup_NO_PICTURE_to_PLACEMENT_with_crit_path(
 
     //Showing given functionality
     crit_path_button_setup(app);
-
+    // hide crit path routing / routing delays
```

Comment:
Why?  Routing data isn't available, so hide the options to display. (say why, not just that you're hiding this).

Suggested comment.

// Routing hasn't been done yet, so hide the display options that show routing as they don't make sense and would crash if clicked on.